Makefile

```diff
@@ -92,7 +92,14 @@ generate-modular-docs: generate-docs ## Generate modular command-line reference
 	SRC_DIR=$$(pwd)/docs/commands DEST_DIR=$$(pwd)/dist go run ./cmd/modular-docs
 .PHONY: generate-modular-docs
 
+lint-lang:
+	git clone git@github.com:redhat-developer/app-services-go-linter
+	cd app-services-go-linter && make binary
+	app-services-go-linter/rhoas-lint
+.PHONY: lint-lang
```

Comment:
It seems a little hacky. It would be better to be able to include it as part of the overall lint configuration. Is that achievable or were there issues?

Reply:
Yes. We should do that in the long run.